### DISCUSSION TITLE
iOS- EXIF Data Sync

### DISCUSSION THREAD

**Discussion by jescott2** [opengisch/QField#3036] [Q&A]
Seems I have no problem taking photos in QField and syncing the photos taken with teh native camera back to my local project on my Windows 10 PC but the EXIF data does not transfer. I have double checked images taken, separately, with the camera app and they do contain the Geotag information. As you can see on one my attachments, if I understand the highlighted icon, the Geotag info was not attached to the photo during capture in QField. I have also included a screen capture of the iPad settings for QField.

System
QField: 2.1.4
iPad 8th Gen
iOS 15.5

Any ideas what to check?

Joel

![D32F90EF-C379-45B2-9CD8-E724FBCFD49D](https://user-images.githubusercontent.com/38323329/176944297-67bfb704-ec73-4110-ab29-25ae4b71bf7b.jpeg)
![QfieldEXIF](https://user-images.githubusercontent.com/38323329/176944356-4e224ade-aaa7-4816-814f-52d59ac4177d.PNG)

**Comment by lucienicolier** [comment]
Hello,

it is currently not possible. iOS is removing exif information from pictures (for privacy reasons) before giving them to apps.

Resolve this is not on our roadmap in short term.
If you want, you can to sponsor this development.
